daemon/src/routes_taker.rs

```diff
@@ -37,11 +36,11 @@ pub async fn feed(
         let order = rx_order.borrow().clone();
         yield order.to_sse_event();
 
-        let cfds = rx_cfds.borrow().clone();
-        yield cfds.to_sse_event();
         let quote = rx_quote.borrow().clone();
         yield quote.to_sse_event();
 
+        let cfds_with_price = CfdsWithCurrentPrice{cfds: rx_cfds.borrow().clone(), current_price: quote.for_taker()};
+        yield cfds_with_price.to_sse_event();
```

Comment:
Instead of doing this, the `maker_cfd::Actor` should also listen for price updates and update its internal state accordingly. It can then send out updates to the frontend.

This will ensure that we are consistent and for example send the correct offer to all connected takers.

Reply:
I've reworked it to make the updates inside  the CFD actors... I don't think it looks much better TBH (perhaps I misunderstood your intention?).

The updates are just for the feed itself, the augmented CFDs were not used within the actor for anything other than for sending the updates to the UI.

Sending the correct offer to the takers is not part of this review, but it indeed would require the maker to listen for the price updates (which is possible with the latest update).

Happy to discuss it tomorrow if I misundestood your comment ;D